vault/resource_identity_entity_alias.go

```diff
@@ -178,3 +189,24 @@ func identityEntityAliasNamePath(name string) string {
 func identityEntityAliasIDPath(id string) string {
 	return fmt.Sprintf("%s/id/%s", identityEntityAliasPath, id)
 }
+
+func findAliasID(client *api.Client, canonicalID, name, mountAccessor string) (string, error) {
+	path := identityEntityIDPath(canonicalID)
+
+	resp, err := client.Logical().Read(path)
+	if err != nil {
+		return "", fmt.Errorf("error reading entity aliases: %s", err)
+	}
+
+	if resp != nil {
```

Comment:
Nit: we could avoid nesting below if we just returned above if response == nil.

Reply:
Yeah, I didn't love it. Chose this route to only have the one instance of the long-ish error message.